From 0bd0cbdf7c170b05c584ab404e4d5487f1363c3d Mon Sep 17 00:00:00 2001
From: "Dark.Hades" <dark.hades.1102@gmail.com>
Date: Mon, 26 Feb 2018 18:21:12 +0900
Subject: [PATCH] Fix error if not in UTF-8 encoding

---
 plugin/NERD_tree.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugin/NERD_tree.vim b/plugin/NERD_tree.vim
index 35b47c3..097d42f 100644
--- a/plugin/NERD_tree.vim
+++ b/plugin/NERD_tree.vim
@@ -69,7 +69,7 @@ call s:initVariable("g:NERDTreeShowHidden", 0)
 call s:initVariable("g:NERDTreeShowLineNumbers", 0)
 call s:initVariable("g:NERDTreeSortDirs", 1)
 
-if !nerdtree#runningWindows() && !nerdtree#runningCygwin()
+if !nerdtree#runningWindows() && !nerdtree#runningCygwin() && &encoding == 'utf-8'
     call s:initVariable("g:NERDTreeDirArrowExpandable", "▸")
     call s:initVariable("g:NERDTreeDirArrowCollapsible", "▾")
 else
-- 
2.16.2

