From 8f00847dce813853efce8a1d17d2672fca446edd Mon Sep 17 00:00:00 2001
From: "Dark.Hades" <dark.hades.1102@gmail.com>
Date: Thu, 1 Mar 2018 16:12:11 +0900
Subject: [PATCH] Fix not in UTF-8 encoding

---
 nerdtree_plugin/git_status.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nerdtree_plugin/git_status.vim b/nerdtree_plugin/git_status.vim
index f0d0981..c67c362 100644
--- a/nerdtree_plugin/git_status.vim
+++ b/nerdtree_plugin/git_status.vim
@@ -42,7 +42,7 @@ if !exists('g:NERDTreeShowIgnoredStatus')
     let g:NERDTreeShowIgnoredStatus = 0
 endif
 
-if !exists('s:NERDTreeIndicatorMap')
+if !exists('s:NERDTreeIndicatorMap') && &encoding == 'utf-8'
     let s:NERDTreeIndicatorMap = {
                 \ 'Modified'  : '✹',
                 \ 'Staged'    : '✚',
-- 
2.16.2

